src/index.js

```diff
@@ -13,13 +13,14 @@ if (process.env.NODE_ENV !== 'production' ) {
 
 module.exports = {
   ...configure,
-  DropdownList:     require('./DropdownList'),
-  Combobox:         require('./Combobox'),
-  Calendar:         require('./Calendar'),
-  DateTimePicker:   require('./DateTimePicker'),
-  NumberPicker:     require('./NumberPicker'),
-  Multiselect:      require('./Multiselect'),
-  SelectList:       require('./SelectList'),
+  Calendar:            require('./Calendar'),
+  Combobox:            require('./Combobox'),
+  DateTimePicker:      require('./DateTimePicker'),
+  DropdownList:        require('./DropdownList'),
+  ListMultiGroupable:  require('./ListMultiGroupable'),
+  Multiselect:         require('./Multiselect'),
+  NumberPicker:        require('./NumberPicker'),
+  SelectList:          require('./SelectList'),
```

Comment:
n/b: Same as above w/ whitespace, if we are going to hack on 3rd party modules we should only do the minimum required changes.